spec/features/user_creates_a_new_review_spec.rb

```diff
@@ -0,0 +1,122 @@
+require "rails_helper"
+
+feature 'user can add a new review for a theme song', %(
+  As a user I can review television program theme song
+  So that I can submit a rating of how much
+  I enjoy it, and optionally a comment on the song!
+) do
+
+  # Acceptance Criteria
+  # [ ] Users must input a rating from 1 to 5
+  # [ ] Users can optionally leave a comment, less than 200 characters
+  # [ ] When the user submits the review, the user review is displayed
+  #     on the show page
+  # [ ] A flash message to indicate success
+  # [ ] A flash message to indicate failure
+
+  scenario 'user adds a review' do
+    user = FactoryGirl.create(:user)
+    program_1 = FactoryGirl.create(:program)
+
+    sign_in(user)
+    expect(page).to have_content('Signed in successfully')
+
+    review = FactoryGirl.create(
+      :review,
+      body: "This is a test! Lets see if it works",
+      program: program_1,
+      user: user
+    )
+
+    visit program_path(program_1)
+    fill_in "Rating", with: review.rating
+    fill_in "Body", with: review.body
+
+    click_button("Add Review")
+    expect(page).to have_content("This is a test! Lets see if it works")
+    expect(page).to have_content(5)
+    expect(page).to have_content(user.email)
+  end
+
+  scenario 'user is not signed in and adds a review' do
+    program_1 = FactoryGirl.create(:program)
+
+    review = FactoryGirl.create(
```

Comment:
I would just manually fill in on lines 51 & 52 versus creating this. Either that or use build instead of create.